processor/src/processors/user_transaction/models/user_transactions.rs

```diff
@@ -171,7 +178,8 @@ pub struct ParquetUserTransaction {
     pub block_timestamp: chrono::NaiveDateTime,
     pub epoch: i64,
     pub sender: String,
-    pub sequence_number: i64,
+    pub sequence_number: Option<i64>,
+    pub nonce: Option<i64>,
```

Comment:
We are using `repaly_protection_nonce` in aptos-core. Should we use the same naming convention here too? I'm fine either ways.